src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -136,7 +136,7 @@ private static object GetNullableAttribute(this MemberInfo memberInfo)
         private static bool GetNullableFallbackValue(this MemberInfo memberInfo)
         {
             var declaringTypes = memberInfo.DeclaringType.IsNested
-                ? new Type[] { memberInfo.DeclaringType, memberInfo.DeclaringType.DeclaringType }
+                ? GetDeclaringTypeChain(memberInfo)
                 : new Type[] { memberInfo.DeclaringType };
```

Comment:
If we changed this to a `List<Type>` then we can avoid the `ToList()` in `GetDeclaringTypeChain()` and just return the list directly.

Reply:
Indeed, fixed